appsDiscovery/src/main/java/org/kinecosystem/appsdiscovery/sender/discovery/presenter/AppsDiscoveryListPresenter.kt

```diff
@@ -1,35 +1,16 @@
 package org.kinecosystem.appsdiscovery.sender.discovery.presenter
 
 import org.kinecosystem.appsdiscovery.base.BasePresenter
-import org.kinecosystem.appsdiscovery.base.OperationResultCallback
 import org.kinecosystem.appsdiscovery.sender.discovery.view.IAppsDiscoveryListView
 import org.kinecosystem.appsdiscovery.sender.model.EcosystemApp
-import org.kinecosystem.appsdiscovery.sender.server.Network
 
 class AppsDiscoveryListPresenter : BasePresenter<IAppsDiscoveryListView>(), IAppsDiscoveryListPresenter {
-    override fun loadData() {
-        val network = Network()
-        network.getDiscoveryApps()
-//        network.getDiscoveryApps(object : OperationResultCallback<List<EcosystemApp>?> {
-//            override fun onResult(apps: List<EcosystemApp>?) {
-//                apps?.let {
-//                    view?.updateData(apps)
-//                }
-//            }
-//
-//            override fun onError(errorCode: Int) {
-//            }
-//        })
+
+    override fun updateData(apps:List<EcosystemApp>) {
+        view?.updateData(apps)
     }
 
     override fun onAppClicked(app: EcosystemApp) {
         view?.navigateToApp(app)
     }
-
-    override fun onAttach(view: IAppsDiscoveryListView) {
-        super.onAttach(view)
-        loadData()
```

Comment:
check that it's on the presenter logic

Reply:
changed the logic of loading is in the presenter